components/PoisList/PoisTable.vue

```diff
@@ -51,67 +51,73 @@ const fields = ref<FieldsListItem[]>()
 const headers = ref<DataTableHeader[]>()
 const category = ref<ApiMenuCategory>()
 
-const { data: pois, error, pending } = useFetch<ApiPois>(
-  `${API_ENDPOINT}/${API_PROJECT}/${API_THEME}/pois/category/${params.id}.geojson`,
+const { data: pois, error, pending, status } = useFetch<ApiPois>(
+  () => `${API_ENDPOINT}/${API_PROJECT}/${API_THEME}/pois/category/${params.id}.geojson`,
   {
     query: {
       geometry_as: 'point',
       short_description: true,
     },
+    transform(pois) {
+      if (!pois.features.length)
+        return pois
+
+      const featureFields = pois.features[0].properties.editorial?.list_fields || []
+
+      // Transform non-string values to single string
+      // Used for sort and filter comparisons
+      return {
+        ...pois,
+        features: pois.features.map((f: ApiPoi) => {
+          const arrayProps: { [key: string]: any } = []
+          const fieldEntries = featureFields.map((f: FieldsListItem) => f.field)
+
+          if (fieldEntries.includes('route'))
+            arrayProps.route = routeToString(f.properties, getContext('route'))
+
+          if (fieldEntries.includes('addr'))
+            arrayProps.addr = addressToString(f.properties)
+
+          return {
+            ...f,
+            properties: {
+              ...f.properties,
+              ...arrayProps,
+            },
+          }
+        }),
+      }
+    },
   },
 )
 
-if (params.id) {
-  category.value = menuStore.getCurrentCategory(Number.parseInt(params.id as string))
-
-  if (!category.value) {
-    throw createError({
-      statusCode: 404,
-      statusMessage: 'Category Not Found',
-    })
+//
+// Watchers
+//
+watch(status, (status) => {
+  switch (status) {
+    case 'error':
+      clearError()
+      break
```

Comment:
missing error report?

Reply:
The error is rendered on UI now, do you think we still need to throw ?
![image](https://github.com/teritorio/vido/assets/38253764/68807b1e-d49a-4bb8-8b41-cb44b60ea3e9)

Reply:
This part is about network error, not http error.